packages/apsara-ui/src/Combobox/Combobox.tsx

```diff
@@ -25,6 +25,7 @@ const Combobox = ({
     showSearch = true,
     showArrow = true,
     placeholder,
+    optionFilterProp = "value",
```

Comment:
this method is actually on works for `undefined`. if user passes `null` or `""` it won't be changed to `"value"`.